string[] name
float64[] confidence
geometry_msgs/PointStamped[] position
